packages/mdc-button/_mixins.scss

```diff
@@ -16,6 +16,7 @@
 
 @import "@material/elevation/mixins";
 @import "@material/ripple/mixins";
+@import "@material/rtl/mixins";
```

Comment:
Add `rtl` to dependencies in button's `package.json`

Reply:
done